index.js

```diff
@@ -73,6 +104,31 @@ function installAddon(data) {
   }, 'application/x-xpinstall');
 }
 
+function getCurrentAddons(cb) {
+  AddonManager.getAllAddons(function(addons) {
+    const ups = updates.map(function(update) {
+                  return update.title.toLowerCase().trim();
+                });
+    cb(addons.filter(function(addonTitle) {
+          return !!~ups.indexOf(addonTitle.name.toLowerCase().trim());
+        }));
+  });
+}
+
+function uninstall(toUninstall, cb) {
+  const ulist = toUninstall.map(function(update) {
+                  return update.toLowerCase().trim();
+                });
+  getCurrentAddons(function(addons) {
+    cb(addons.filter(function(a) {
+          return !!~ulist.indexOf(a.name.toLowerCase().trim());
```

Comment:
Not sure if it's worth extracting the `.toLowerCase().trim()` bit out into a separate `normalize()` function (since it looks like we're using it 6 times in this file).

Similarly, if there is a way to "pretty up" the double-negation bitwise op, apart from adding some **lodash** dependency and possibly using something like:
```js
cb(addons.filter(function(a) {
  return _.contains(ulist, normalize(a.name));
}) ...
```